pinot-common/src/main/java/org/apache/pinot/common/utils/config/QueryOptionsUtils.java

```diff
@@ -145,6 +145,11 @@ public static boolean isSkipScanFilterReorder(Map<String, String> queryOptions)
     return "false".equalsIgnoreCase(queryOptions.get(QueryOptionKey.USE_SCAN_REORDER_OPTIMIZATION));
   }
 
+  @Nullable
+  public static boolean isUseStickyRouting(Map<String, String> queryOptions) {
```

Comment:
We should return `Boolean` here, and differentiate not set and explicit false

Reply:
I updated to use queryOption -> broker config fallback